.github/workflows/ci.yaml

```diff
@@ -255,6 +255,11 @@ jobs:
         run: |
           ./project/scripts/sbt ";scala3-interfaces/mimaReportBinaryIssues ;scala3-library-bootstrapped/mimaReportBinaryIssues ;scala3-library-bootstrappedJS/mimaReportBinaryIssues; tasty-core-bootstrapped/mimaReportBinaryIssues; stdlib-bootstrapped/mimaReportBinaryIssues"
 
+      - name: TASTy MiMa
```

Comment:
This takes around 5 minutes. An alternative to this would be to create a new ci job that only executes on nightly or with a flag. Currently, I am the only one doing changes that require these tests and it might not be worth on each PR.